syntax = "proto3";
import "public.proto";
package fixbug;

option cc_generic_services = true;


message GetFriendsListRequest
{
    int32 userid = 1;
}
message Friends{
    int32 userid = 1;
    bytes friendname =2;
    bytes online = 3;
}

message GetFriendsListResponse
{
    ResultCode result = 1;
    repeated Friends friends = 2;
}


// 好友模块
service FiendServiceRpc
{
    rpc GetFriendsList(GetFriendsListRequest) returns(GetFriendsListResponse);
}
